Dockerfile

```diff
@@ -60,25 +60,25 @@ RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php -y && \
     apt-get update && \
     apt-get install -y \
         unzip \
-        php8.1-cli \
-        php8.1-dev \
-        php8.1-common \
-        php8.1-zip \
-        php8.1-xml \
-        php8.1-pgsql \
-        php8.1-curl \
-        php8.1-imagick \
-        php8.1-imap \
-        php8.1-intl \
-        php8.1-sqlite3 \
-        php8.1-bcmath \
-        php8.1-soap \
-        php8.1-mysql \
-        php8.1-gd \
-        php8.1-xsl \
-        php8.1-sysvsem \
-        php8.1-mbstring \
-        php8.1-xdebug \
+        php8.2-cli \
+        php8.2-dev \
+        php8.2-common \
+        php8.2-zip \
+        php8.2-xml \
+        php8.2-pgsql \
+        php8.2-curl \
+        php8.2-imagick \
+        php8.2-imap \
+        php8.2-intl \
+        php8.2-sqlite3 \
+        php8.2-bcmath \
+        php8.2-soap \
+        php8.2-mysql \
+        php8.2-gd \
+        php8.2-xsl \
+        php8.2-sysvsem \
+        php8.2-mbstring \
```

Comment:
the image is supposed to run only apps that access a postgresql database and nothing more, so remove all the unecessary php dependencies